ydb/docs/en/core/yql/reference/yql-core/syntax/alter-topic.md

```diff
@@ -141,3 +141,31 @@ ALTER TOPIC `my_topic` RESET (
 * `supported_codecs`: List of [codecs](../../../../concepts/topic#message-codec) supported by the topic. Value type: `String`.
 
 {% endif %}
+
+### Change autopartitioning modes for the topic {#autopartitioning}
+
+Next command set [autopartitioning](../../../../concepts/topic#autopartitioning) into the mode UP:
+
+```yql
+ALTER TOPIC `my_topic` SET (
+    min_active_partitions = 1,
+    max_active_partitions = 5,
+    auto_partitioning_strategy = 'scale_up'
```

Comment:
Two naming inconsistencies:

* above, we call these "modes", while the actual setting name is "strategy"
* above, we call this mode/strategy just "up", while the actual setting value is "scale up"

Reply:
1. I'll change "mode" into the "strategy"? 
2. YQL will be fixed here, and after that both "scale up" & "up" will be possible, and we'll change the documentation after that (not in this PR).